### PR TITLE
fix errors and add how to publish ts package

### PR DESCRIPTION
**Pull request by avalynndev**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
 * @EasyScriptJS/maintainers
+* @avalynndev
```
Comment (wdhdev) [inline .github/CODEOWNERS]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - run:
+          name: Generate package-lock.json
+          command: npm i --package-lock-only
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          name: Intall TypeScript globally
+          command: sudo npm i -g typescript
+      - run:
+          name: Compile test-avalynn
```
Comment (wdhdev) [inline .circleci/config.yml]:
```suggestion
          name: Compile
```

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+	"trailingComma": "none",
+	"tabWidth": 2,
```
Comment (wdhdev) [inline .prettierrc.json]:
```suggestion
	"tabWidth": 4,
```

File: NOTE_FOR_DEVS.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+## To publish the package:
```
Comment (wdhdev) [inline NOTE_FOR_DEVS.md]: Don't need this file as it is automated.

File: README.md [hunk 5]
```diff
@@ -1,18 +1,93 @@
 <p align="center">
-  <img src="https://images.easyscript.dev/cover.png">
+  <img src="https://images.easyscript.dev/banner.png" />
```
Comment (wdhdev) [inline README.md]:
```suggestion
  <img src="https://images.easyscript.dev/banner.png">
```

File: README.md [hunk 6]
```diff
@@ -1,18 +1,93 @@
...
+  <img alt="Status" src="https://img.shields.io/badge/status-active-success.svg?style=for-the-badge" />
+  <img alt="GitHub Issues" src="https://img.shields.io/github/issues/EasyScriptJS/EasyScript.svg?style=for-the-badge" />
+  <img alt="Pull Requests" src="https://img.shields.io/github/issues-pr/EasyScriptJS/EasyScript.svg?style=for-the-badge" />
+  <img alt="License" src="https://img.shields.io/badge/license-%20Creative%20Commons-blue?style=for-the-badge" />
 </p>
 
-<p align="center">Easy Script is a npm package which makes coding in JavaScript easy!</p>
+---
+
+# 🔻 Easy Script.js<a name = "about"></a>
```
Comment (wdhdev) [inline README.md]:
```suggestion
# 📄 Easy Script
```

File: README.md [hunk 7]
```diff
@@ -1,18 +1,93 @@
...
+
+Easy Script is a npm package which makes coding in JavaScript easy!
+
+<br />
```
Comment (wdhdev) [inline README.md]:
```suggestion
<br>
```

File: README.md [hunk 8]
```diff
@@ -1,18 +1,93 @@
...
-<h3 align="center"><code>npm install easyscriptjs</code></p></h3>
-
 <p align="center">
-  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge">
-  <img alt="GitHub Issues" src="https://img.shields.io/github/issues-raw/easyscriptjs/easyscript?label=Issues&style=for-the-badge">
-  <img alt="GitHub Pull Requests" src="https://img.shields.io/github/issues-pr-raw/easyscriptjs/easyscript?label=Pull%20Requests&style=for-the-badge">
+  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge" />
+  <img alt="Status" src="https://img.shields.io/badge/status-active-success.svg?style=for-the-badge" />
+  <img alt="GitHub Issues" src="https://img.shields.io/github/issues/EasyScriptJS/EasyScript.svg?style=for-the-badge" />
+  <img alt="Pull Requests" src="https://img.shields.io/github/issues-pr/EasyScriptJS/EasyScript.svg?style=for-the-badge" />
```
Comment (wdhdev) [inline README.md:8]: Please keep the original PRs badge.

File: README.md [hunk 9]
```diff
@@ -1,18 +1,93 @@
...
+  <img alt="License" src="https://img.shields.io/badge/license-%20Creative%20Commons-blue?style=for-the-badge" />
```
Comment (wdhdev) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -1,18 +1,93 @@
...
-
-<h3 align="center"><code>npm install easyscriptjs</code></p></h3>
-
 <p align="center">
-  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge">
-  <img alt="GitHub Issues" src="https://img.shields.io/github/issues-raw/easyscriptjs/easyscript?label=Issues&style=for-the-badge">
-  <img alt="GitHub Pull Requests" src="https://img.shields.io/github/issues-pr-raw/easyscriptjs/easyscript?label=Pull%20Requests&style=for-the-badge">
+  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge" />
+  <img alt="Status" src="https://img.shields.io/badge/status-active-success.svg?style=for-the-badge" />
+  <img alt="GitHub Issues" src="https://img.shields.io/github/issues/EasyScriptJS/EasyScript.svg?style=for-the-badge" />
```
Comment (wdhdev) [inline README.md:7]: Please keep original Issues badge.

File: README.md [hunk 11]
```diff
@@ -1,18 +1,93 @@
...
-<h1 align="center">Easy Script</h1>
-
-<h3 align="center"><code>npm install easyscriptjs</code></p></h3>
-
 <p align="center">
-  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge">
-  <img alt="GitHub Issues" src="https://img.shields.io/github/issues-raw/easyscriptjs/easyscript?label=Issues&style=for-the-badge">
-  <img alt="GitHub Pull Requests" src="https://img.shields.io/github/issues-pr-raw/easyscriptjs/easyscript?label=Pull%20Requests&style=for-the-badge">
+  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge" />
+  <img alt="Status" src="https://img.shields.io/badge/status-active-success.svg?style=for-the-badge" />
```
Comment (wdhdev) [inline README.md]:
```suggestion
```
Not needed.

File: README.md [hunk 12]
```diff
@@ -1,18 +1,93 @@
...
 
-<h1 align="center">Easy Script</h1>
-
-<h3 align="center"><code>npm install easyscriptjs</code></p></h3>
-
 <p align="center">
-  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge">
-  <img alt="GitHub Issues" src="https://img.shields.io/github/issues-raw/easyscriptjs/easyscript?label=Issues&style=for-the-badge">
-  <img alt="GitHub Pull Requests" src="https://img.shields.io/github/issues-pr-raw/easyscriptjs/easyscript?label=Pull%20Requests&style=for-the-badge">
+  <img alt="Lastest Release" src="https://img.shields.io/github/v/release/easyscriptjs/easyscript?style=for-the-badge" />
```
Comment (wdhdev) [inline README.md:6]: Please keep original one.

File: README.md [hunk 13]
```diff
@@ -1,18 +1,93 @@
...
+<br />
+
+## 📝 Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Built Using](#built_using)
+- [Authors](#authors)
+
+<br />
```
Comment (wdhdev) [inline README.md]:
```suggestion
<br>
```

File: README.md [hunk 14]
```diff
@@ -1,18 +1,93 @@
...
+
+## 🏁 Getting Started <a name = "getting_started"></a>
+
+These instructions will get yuo ready for using easyscript. See [docs](https://docs.easyscript.dev) on how to use this package to make your work faster.
+
+<br />
+
+## 🖥️ How to install ?
```
Comment (wdhdev) [inline README.md:19]: Please remove this section

File: README.md [hunk 15]
```diff
@@ -1,18 +1,93 @@
...
+## 📝 Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Built Using](#built_using)
+- [Authors](#authors)
+
+<br />
+
+## 🏁 Getting Started <a name = "getting_started"></a>
```
Comment (wdhdev) [inline README.md]: Please remove

File: README.md [hunk 16]
```diff
@@ -1,18 +1,93 @@
...
+
+```bash
+$> Hello World
+```
+
+See [docs](https://docs.easyscript.dev) on how to use this package's functions and make your work faster.
+
+<br />
+
+## 🤔 But Why ?
```
Comment (wdhdev) [inline README.md]:
```suggestion
## 🤔 Why?
```

File: README.md [hunk 17]
```diff
@@ -1,18 +1,93 @@
...
+- Beginner Friendly
+- Open **[Source project](https://github.com/EasyScriptJS/EasyScript)** !
+- Simple & Understandable **[Documentation](https://docs.easyscript.dev)**
+
+<br />
+
+## ⛏️ Built Using <a name = "built_using"></a>
+
+- [Chalk](https://www.mongodb.com/) - Terminal string styling done right
+- [Typescript](https://www.typescriptlang.org) - Programming Language
```
Comment (wdhdev) [inline README.md]:
```suggestion
- [TypeScript](https://www.typescriptlang.org) - Programming Language
```

File: README.md [hunk 18]
```diff
@@ -1,18 +1,93 @@
...
+- [NodeJS](https://nodejs.org/en/) - Server Environment
```
Comment (wdhdev) [inline README.md]:
```suggestion
- [Node.js](https://nodejs.org) - Server Environment
```

File: README.md [hunk 19]
```diff
@@ -1,18 +1,93 @@
...
+- Easier yet Flexible.
+- Beginner Friendly
+- Open **[Source project](https://github.com/EasyScriptJS/EasyScript)** !
+- Simple & Understandable **[Documentation](https://docs.easyscript.dev)**
+
+<br />
+
+## ⛏️ Built Using <a name = "built_using"></a>
+
+- [Chalk](https://www.mongodb.com/) - Terminal string styling done right
```
Comment (wdhdev) [inline README.md]: I think you're using the wrong URL here...

File: README.md [hunk 20]
```diff
@@ -1,18 +1,93 @@
...
+- [Typescript](https://www.typescriptlang.org) - Programming Language
+- [NodeJS](https://nodejs.org/en/) - Server Environment
+
+<br />
```
Comment (wdhdev) [inline README.md]:
```suggestion
<br>
```

File: README.md [hunk 21]
```diff
@@ -1,18 +1,93 @@
...
+
+## ✍️ Authors <a name = "authors"></a>
+
+- [@WilliamDavidHarrison](https://github.com/WilliamDavidHarrison) - Idea & Initial work
+- [@avalynndev](https://github.com/avalynndev) - converting to typescript, documentation
```
Comment (wdhdev) [inline README.md]:
```suggestion
- [@avalynndev](https://github.com/avalynndev) - Docs Team
```

File: README.md [hunk 22]
```diff
@@ -1,18 +1,93 @@
...
 
-<h2 align="center">Documentation</h2>
-<p align="center">If you want to learn how to use Easy Script, you can find all of the information you need on the <a href="https://docs.easyscript.dev">docs</a>.</p>
+See also the list of [contributors](https://github.com/EasyScriptJS/EasyScript/contributors) who participated in this project.
```
Comment (wdhdev) [inline README.md]:
```suggestion
See also the list of [contributors](https://github.com/EasyScriptJS/EasyScript/contributors) who have helped build this project.
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by wdhdev** [comment]
Please also fix merge conflicts

File: README.md (hunk 19 above)
Comment (avalynndev) [reply README.md]: ya
ima fix it

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please revert the license change.

File: .prettierrc.json (hunk 3 above)
Comment (wdhdev) [reply .prettierrc.json]: I prefer 4.

**Comment by wdhdev** [comment]
I'm going to close this PR as it makes too many unnecessary changes. Please open a new PR **only updating the source code**. Please also make sure the base branch is pointing to the **release/v2.0.0** branch. Also, please use the PR template.

**Comment by avalynndev** [comment]
bruh

**Pull request closed without merging**